app/models/account/credit_card.rb

```diff
@@ -134,18 +131,26 @@ def notify_credit_card_change
 
   private
 
+  delegate :payment_gateway_setting_exists?, to: :provider_account, prefix: true, allow_nil: true
+
   # voids the transaction with the +authorization+ code
   def void_transaction!(authorization)
     response = provider_payment_gateway.void(authorization)
     log_gateway_response(response, "void [transaction: #{authorization}]")
     response.success?
   end
 
+  def threescale_unstore_payment_gateway
```

Comment:
I think there's a confusion of terminology here. There's no "unstore payment gateway", we unstore "payment details", tokenized credit cards in the payment gateway.